### PR TITLE
Cluster and project logging data sources

### PR DESCRIPTION
**Pull request by rawmind0**
Fix issue #11

### CODE REVIEW

**Comment by cloudnautique** [comment]
Do we need to mark the `*_configs` as sensitive on the data sources since the config might have username/passwords?

**Comment by rawmind0** [comment]
I think it's not needed, due to username/passwords are already marked as sensitive on `logging*ConfigFields()` definitions.

File: website/docs/d/projectLogging.html.markdown [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  project_id = "<project_id>"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `project_id` - (Required) The project id to configure logging (string)
+* `name` - (Required) The name of the cluster logging config (string)
```
Comment (cloudnautique) [inline website/docs/d/projectLogging.html.markdown]: There is on only one per project, and cluster. Why do they need the name? Type maybe...

Comment (rawmind0) [reply website/docs/d/projectLogging.html.markdown]: You are right. Addressing it to let name as computed field.

Comment (rawmind0) [reply website/docs/d/projectLogging.html.markdown]: Fixed `name` argument as computed.

File: website/docs/d/projectLogging.html.markdown [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+# rancher2\_project\_logging
+
+Use this data source to retrieve information about a Rancher v2 Project Logging.
+
+## Example Usage
+
+```hcl
+data "rancher2_project_logging" "foo" {
+  name = "foo"
```
Comment (cloudnautique) [inline website/docs/d/projectLogging.html.markdown]: Name is referenced here.

File: website/docs/d/clusterLogging.html.markdown [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+# rancher2\_cluster\_logging
+
+Use this data source to retrieve information about a Rancher v2 Cluster Logging.
+
+## Example Usage
+
+```hcl
+data "rancher2_cluster_logging" "foo" {
+  name = "foo"
```
Comment (cloudnautique) [inline website/docs/d/clusterLogging.html.markdown]: is this no longer needed now?

File: website/docs/d/projectLogging.html.markdown (hunk 2 above)
Comment (rawmind0) [reply website/docs/d/projectLogging.html.markdown]: Addressed.

File: website/docs/d/clusterLogging.html.markdown (hunk 3 above)
Comment (rawmind0) [reply website/docs/d/clusterLogging.html.markdown]: Addressed.

**Review by cloudnautique — APPROVED** [review]

**Pull request merged**